.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+---
+version: 2.1
+jobs:
+  build:
```

Comment:
Just a heads up, it doesn't need to be named "build" - you can give it an easily identifyable name, which will also show up as this on github. If you are interested in adding more jobs on circleci.

See e.g. https://github.com/naxuroqa/Venom/blob/develop/.circleci/config.yml#L103

Reply:
Thanks for the hints, I think I'll consult with you before I make additional updates to our CircleCI system.